docs/userguide/eng-image/dockerfile_best-practices.md

```diff
@@ -142,7 +173,7 @@ As always, to make your `Dockerfile` more readable, understandable, and
 maintainable, split long or complex `RUN` statements on multiple lines separated
 with backslashes.
 
-### apt-get
+#### apt-get
```

Comment:
This was incidental, but it seemed like if RUN was a third-level heading, and apt-get was a RUN example, it should be a 4th-level heading. That's why the change is here even though it is not really relevant to this PR.